content/registration.md

```diff
@@ -8,6 +8,19 @@ weight: 2
 
 The registration will open after the preliminaries have taken place. 
 
-The slots per institution for BAPC21 are limited. As there tends to be more teams than slots available, your institution can hold preliminary contests to determine which teams qualify for BAPC21. The preliminaries are expected to be held in the summer of 2021. 
+## Preliminaries
+
+The slots per institution for BAPC21 are limited. As there tends to be more teams than slots available, your institution can hold preliminary contests to determine which teams qualify for BAPC21. The preliminaries will hold on **Saturday, 9 October**.
```

Comment:
```suggestion
The slots per institution for BAPC21 are limited. As there tends to be more teams than slots available, your institution can hold preliminary contests to determine which teams qualify for BAPC21. The preliminaries should hold on **Saturday, 9 October**.
```

Reply:
Will you be okay with "planned for Saturday, 9 October"?

Reply:
Yes I think thats fine.

Or not mention it here as I prefer to have 1 schedule page. Those mentions can be misleading if we forget to update them. The people registering will know of such things. But if you want to keep the date there than the formulation is fine by me.